code/signup.html

```diff
@@ -0,0 +1,126 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Project 3</title>
+
+  <!-- Google font import -->
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Montserrat&display=swap" rel="stylesheet">
+
+  <!-- CSS Import -->
+  <link rel="stylesheet" href="./style.css">
+
+</head>
+
+<body>
+
+  <!-- Header container start -->
+  <div class="headerContainer">
+
+    <div class="navigation">
+
+      <div class="TabletDesktopMenu">
+        <a class="menuLink" href="./index.html">Home</a>
+        <a class="menuLink" href="./signup.html">Sign up</a>
+      </div>
+      <!-- Mobile Hamburger menu start -->
+      <div>
+        <button class="ham"></button>
+        <nav class="navbar">
+          <ul>
+            <li><a class="menuLink" href="./index.html">Home</a></li>
+            <li><a class="menuLink" href="./signup.html">Sign up</a></li>
+          </ul>
+        </nav>
```

Comment:
Another option to clean up the structure a bit is to omit the `ul` and the `li` elements, and style the `a` tags as list items.

Reply:
Thanks for the tip! :)